setup.py

```diff
@@ -96,6 +96,7 @@
             "isort>=5.12.0",
             "daphne>=4.0.0",
             "django-stubs>=4.2.4",
+            "djangorestframework-stubs[compatible-mypy]>=3.14.2",
```

Comment:
This version should probably be the same as `djangorestframework`, so `~=3.14.0`.

Reply:
Not sure: `djangorestframework` has no `3.14.2` release, but unifying them will not hurt.